src/ucx_van.h

```diff
@@ -209,8 +209,12 @@ class UCXEndpointsPool {
     void *req = ucp_ep_close_nb(ep, UCP_EP_CLOSE_MODE_FLUSH);
     if (UCS_PTR_IS_PTR(req)) {
       close_ep_reqs_.push(req);
-    } else if (UCS_PTR_STATUS(req) != UCS_OK) {
-      LOG(ERROR) << "failed to close ep: " << ep;
+    } else {
+      ucs_status_t status = UCS_PTR_STATUS(req);
+      if ((status != UCS_OK) && (status != UCS_ERR_ENDPOINT_TIMEOUT)) {
```

Comment:
does this mean a `UCS_ERR_ENDPOINT_TIMEOUT ` is always from calling `ucp_ep_close_nb` on an unconnected endpoint?

Reply:
for now yes, let's suppress this error